src/models/branches/lines.jl

```diff
@@ -3,38 +3,52 @@ const From_To_Float = NamedTuple{(:from, :to),Tuple{Float64,Float64}}
 const From_To_Bus =  NamedTuple{(:from, :to),Tuple{Bus,Bus}}
 const FromTo_ToFrom_Float = NamedTuple{(:from_to, :to_from),Tuple{Float64,Float64}}
 
-
 struct Line <: Branch
     name::String
     available::Bool
     connectionpoints::From_To_Bus
     r::Float64 #[pu]
     x::Float64 #[pu]
     b::From_To_Float # [pu]
-    rate::FromTo_ToFrom_Float #MW
+    rate::Float64
     anglelimits::Min_Max #Degrees
 end
 
 function Line(name::String, available::Bool, connectionpoints::From_To_Bus,
-              r::Float64, x::Float64, b::From_To_Float, rate::FromTo_ToFrom_Float, anglelimits::Float64)
```

Comment:
Do we need to remove the directional flow limit definition? If so, fine... If not, I like having this feature.

Reply:
The idea is to separate the notion of the rate which the inherent physical limitations of the line from the flow_limits from other constraints. This will remove the definition conflict with PowerModels.